torch/jit/__init__.py

```diff
@@ -1386,6 +1386,7 @@ def register_all(mod):
     _builtin_table[id(_list_with_default)] = "aten::list_with_default"
     _builtin_table[id(_unwrap_optional)] = "aten::_unwrap_optional"
     _builtin_table[id(cudnn.is_acceptable)] = "aten::cudnn_is_acceptable"
+    _builtin_table[id(torch._C._infer_size)] = "aten::_infer_size"
```

Comment:
Could you just write at::infer_size and not add the extra operator ?

Reply:
There's no `aten` op for `infer_size` (plus we have 2 overloads of `infer_size` in `ExpandUtils.cpp` and in `InferSize.h`), so I think we have to have an op for it